engines/dreamweb/dreamweb.cpp

```diff
@@ -392,6 +394,27 @@ void DreamWebEngine::processEvents(bool processSoundEvents) {
 }
 
 Common::Error DreamWebEngine::run() {
+	Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
+	if (ttsMan != nullptr) {
+		switch(getLanguage()) {
+		case Common::ES_ESP:
+			ttsMan->setLanguage("es");
+			break;
+		case Common::FR_FRA:
+			ttsMan->setLanguage("fr");
+			break;
+		case Common::IT_ITA:
+			ttsMan->setLanguage("it");
+			break;
+		case Common::DE_DEU:
+			ttsMan->setLanguage("de");
+			break;
+		default:
+			ttsMan->setLanguage("en");
+			break;
+		}
+	}
+
```

Comment:
Does that work when the language is Russian?

Also you can probably use `Common::getLanguageCode` instead to simplify the code:
```c++
ttsMan->setLanguage(Common::getLanguageCode(getLanguage()));
```